.changeset/strange-rocks-poke.md

```diff
@@ -0,0 +1,5 @@
+---
+"hardhat": patch
```

Comment:
Should this be `@hardhat-core`? @fvictorio

Reply:
No, `hardhat` is fine. `packages/hardhat-core` is named like that to make talking about `hardhat` (the npm package) simpler